showcase/src/app/home/home.component.scss

```diff
@@ -0,0 +1,56 @@
+.documentation-links {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: center;
+  align-items: center;
+  & a {
+    font-size: 18px;
+    text-decoration: none;
+    text-align: center;
+  }
+}
+
+.documentation-title {
+  text-align: center;
+}
+
+.stark-core-latest {
+  background-image: url("/assets/images/logo/logo-blue.svg");
+  background-size: cover;
+  height: 100px;
+  width: 100px;
+  margin: 0 1em;
+}
+
+.stark-core {
+  background-image: url("/assets/images/logo/core.png");
+  background-size: cover;
+  text-align: center;
+  height: 100px;
+  width: 100px;
+  margin: 0 1em;
```

Comment:
You can refactor these styles so that the common styles of all images (text-align, height, width, margin) are defined only once instead of repeating them every time ;)